src/objects/_main-wrapper.scss

```diff
@@ -19,6 +19,8 @@
 @mixin govuk-main-wrapper {
   @include govuk-responsive-padding(6, "top");
   @include govuk-responsive-padding(6, "bottom");
+  // IE11 supports `main` but doesn't set it to block without HTML5shiv
```

Comment:
As far as I know html5shiv isn't involved here - IE11 does support the `<main>` element 'natively', it's just not styled correctly. The shiv is only invoked for Internet Explorer 6-9 according to their readme.